azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerPipelineTest.java

```diff
@@ -67,7 +65,7 @@
  *
  * @author rpark
  */
-public class FlowRunnerPipelineTest {
+public class FlowRunnerPipelineTest extends FlowRunnerTestBase {
```

Comment:
not part of this PR, but I think FlowRunnerTestBase.java needs to be documented better and it's missing license.